homeassistant/components/bayesian/strings.json

```
@@ -14,5 +14,192 @@
       "name": "[%key:common::action::reload%]",
       "description": "Reloads Bayesian sensors from the YAML-configuration."
     }
+  },
+  "options": {
+    "error": {
+      "extreme_prior_error": "[%key:component::bayesian::config::error::extreme_prior_error%]",
+      "extreme_threshold_error": "[%key:component::bayesian::config::error::extreme_threshold_error%]",
+      "equal_probabilities": "[%key:component::bayesian::config::error::equal_probabilities%]",
+      "extreme_prob_given_error": "[%key:component::bayesian::config::error::extreme_prob_given_error%]"
+    },
+    "step": {
+      "init": {
+        "title": "Sensor options",
+        "description": "These options affect how much evidence is required for the Bayesian sensor to be considered 'on'.",
+        "data": {
+          "probability_threshold": "[%key:component::bayesian::config::step::user::data::probability_threshold%]",
+          "prior": "[%key:component::bayesian::config::step::user::data::prior%]",
+          "device_class": "[%key:component::bayesian::config::step::user::data::device_class%]",
+          "name": "[%key:common::config_flow::data::name%]"
+        },
+        "data_description": {
+          "probability_threshold": "[%key:component::bayesian::config::step::user::data_description::probability_threshold%]",
+          "prior": "[%key:component::bayesian::config::step::user::data_description::prior%]"
+        }
+      }
+    }
+  },
+  "config": {
+    "error": {
+      "extreme_prior_error": "A prior set to 0% means that it is impossible for the sensor to show 'on' and 100% means it will never show off, use a close number like 0.1% or 99.9% instead",
+      "extreme_threshold_error": "A threshold set to 0% means that the sensor will always be 'on' and 100% mean it will always be off, use a close number like 0.1% or 99.9% instead",
```

Here similar, when referring to "Probability threshold":

```suggestion
      "extreme_threshold_error": "'Probability threshold' set to 0% means that the sensor will always be 'on' and 100% mean it will always be 'off', use a close number like 0.1% or 99.9% instead",
```